week2/homework/src/data.json

```diff
@@ -0,0 +1,6 @@
+{
+    "todo": {
+        "1": "shopping",
```

Comment:
When you are making an object, it makes more sense to use something like
```
{
    "id": 1,
    "text": "shopping"
}
```
That way you can use `todo.id` and `todo.text`. If you use the id as key, it is actually similar to an array.

Reply:
thanks for this helpful feedbacks , all is done !